.storybook/preview.ts

```diff
@@ -1,8 +1,12 @@
-import type { Preview } from "@storybook/web-components";
+import type {Preview} from '@storybook/web-components';
```

Comment:
Adds web components support on **StoryBook** so it can consume the `custom-elements.json` manifest